xgi/convert.py

```diff
@@ -302,3 +306,103 @@ def to_incidence_matrix(H, sparse=True, index=False):
     from_incidence_matrix
     """
     return xgi.incidence_matrix(H, sparse=sparse, index=index)
+
+
+def from_bipartite_network(G, create_using=None, dual=False):
+    """
+    Create a Hypergraph from a NetworkX bipartite graph.
```

Comment:
What about adding a Ref. to Leo's SIAM Rev. on representations?

Reply:
Done.